src/EventSubscriber/CreateOAuthUserSubscriber.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the user bundle package.
+ * (c) Connect Holland.
+ */
+
+namespace ConnectHolland\UserBundle\EventSubscriber;
+
+use ConnectHolland\UserBundle\Event\CreateOAuthUserEventInterface;
+use ConnectHolland\UserBundle\UserBundleEvents;
+
+final class CreateOAuthUserSubscriber implements CreateOAuthUserSubscriberInterface
+{
+    public function onCreateOAuthUser(CreateOAuthUserEventInterface $event): void
+    {
+        $user = $event->getUser();
+        $user->setEnabled(true);
+        if ($event->getResponse()->getEmail()) {
+            $user->setEmail($event->getResponse()->getEmail());
+        }
+    }
+
+    /**
+     * @codeCoverageIgnore No need to test this array 'config' method
+     */
+    public static function getSubscribedEvents()
+    {
+        return [
+            UserBundleEvents::CREATE_OAUTH_USER => 'onCreateOAuthUser',
+        ];
+    }
```

Comment:
Shouldn't statics be placed above non-static methods? Especially in eventSubscribers it makes sense to me as it is somewhat of an index of the class

Reply:
This is the default that symfony maker bundle generates, I think it suits the usecase here. Do you agree?